lib/acts_as_tenant.rb

```diff
@@ -14,6 +13,10 @@ module ActsAsTenant
   @@models_with_global_records = []
   @@mutable_tenant = false
 
+  class Current < ActiveSupport::CurrentAttributes
+    attribute :current_tenant, :test_tenant, :acts_as_tenant_unscoped
```

Comment:
@pond you mean this?

```suggestion
    attribute :current_tenant, :acts_as_tenant_unscoped
```

Reply:
Yep, exactly that 👍